Formula/cryptominisat.rb

```diff
@@ -32,10 +32,13 @@ def install
     # fix audit failure with `lib/libcryptominisat5.5.7.dylib`
     inreplace "src/GitSHA1.cpp.in", "@CMAKE_CXX_COMPILER@", ENV.cxx
 
-    mkdir "build" do
-      system "cmake", "..", *std_cmake_args, "-DNOM4RI=ON", "-DCMAKE_INSTALL_RPATH=#{rpath}"
-      system "make", "install"
-    end
+    system "cmake", "-S", ".", "-B", "build",
+                    "-DNOM4RI=ON",
+                    "-DCMAKE_INSTALL_RPATH=#{rpath}",
+                    "-DCMAKE_CXX_COMPILER=#{ENV.cxx} -std=gnu++11",
```

Comment:
```suggestion
                    "-DCMAKE_CXX_COMPILER=#{ENV.cxx}",
```
The build complains about this not being a path.

Reply:
Sigh, CMake is so annoying